doc/examples/edges/plot_edge_filter.py

```diff
@@ -56,35 +54,36 @@
 # .. [3] https://en.wikipedia.org/wiki/Prewitt_operator
 
 x, y = np.ogrid[:100, :100]
-# Rotation-invariant image with different spatial frequencies
-img = np.exp(1j * np.hypot(x, y) ** 1.3 / 20.).real
 
-edge_sobel = sobel(img)
-edge_scharr = scharr(img)
-edge_prewitt = prewitt(img)
+# Creating a rotation-invariant image with different spatial frequencies.
+image_rot = np.exp(1j * np.hypot(x, y) ** 1.3 / 20.).real
+
+edge_sobel = filters.sobel(image_rot)
+edge_scharr = filters.scharr(image_rot)
+edge_prewitt = filters.prewitt(image_rot)
 
 diff_scharr_prewitt = edge_scharr - edge_prewitt
```

Comment:
Can't we use https://scikit-image.org/docs/stable/api/skimage.util.html#compare-images over here?